### PR TITLE
Update analysis_differential_expression_help.md

### PR DESCRIPTION
**Pull request by jennaj**
Updated based on more recent followup. Added a tool. More cross-links.

### CODE REVIEW

**Review by jennaj — COMMENTED** [review]
Looks good not sure what problem is.

File: faqs/galaxy/analysis_differential_expression_help.md [hunk 1]
```diff
@@ -7,34 +7,69 @@ layout: faq
...
+Reference transcriptome (fasta)
+
+- Fasta file containing assembled transcripts.
+- Unassembled short or long reads will not work as a substitute.
+- The transcript identifiers on the `>seq` fasta lines must exactly match the `transcript_id` values in your annotation or tabular mapping file.
+- Sometimes **Salmon** or **DESeq2** (when comparing TMP values) does not understand `transcript_id.N` (where N is a version number). Try removing `.N` from all inputs. 
+  
+Reference annotation (tabular, GTF, GFF3)
+
+- Reference annotation [in GTF format]({% link faqs/galaxy/datasets_working_with_reference_annotation.md & %}) works best.
```
[the remark this answers is not in the document]
Comment (jennaj) [reply faqs/galaxy/analysis_differential_expression_help.md:65]: Where is the & coming from?

**Review by jennaj — COMMENTED** [review]
Actually good now

**Review by jennaj — COMMENTED** [review]
Good

**Comment by jennaj** [comment]
@hexylena Asking for your review since I changed it up so much. But think changes will help users more than the original. Mostly want to double check that I didn't introduce a problem. The # char was the problem, yes?

And -- can I put multiple FAQs in a single branch or do you want them separate? Thx

**Comment by hexylena** [comment]
> And -- can I put multiple FAQs in a single branch or do you want them separate? Thx

multiple in a single branch is totes ok!!! easier to review :)

> the & char

yeah `{% link ... %}` expects the rest of that link to be an exact filename, if you need URL parameters please put them after, e.g. `{% link page.md %}?x=y`

**Review by hexylena — APPROVED** [review]
This looks like a fantastic improvement, so much cleaner and easier to read, and more links is a great bonus!

**Pull request merged**